LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020 Raphael Robert
+Copyright (c) 2022 OpenMLS Authors
```

Comment:
```suggestion
Copyright (c) 2020 OpenMLS Authors
```

Reply:
My understanding is that the year in the copyright notice reflects the creation time of the work. Since we have continually updated the code until now, wouldn't the year be 2022?